.travis.yml

```diff
@@ -78,6 +81,11 @@ jobs:
       skip_cleanup: true
       on:
         all_branches: true  # but see the `if:` conditional on the deploy stage above
+  - stage: deploy-dev
+    script:
+    - "http POST ${GITLAB_API}/projects/${REPO_SLUG}/mirror/pull Private-Token:${GITLAB_TOKEN}"
+    - sleep 60  # give the build box some breathing room to update repo
```

Comment:
Is there any way we can use some sort of polling here? Or trigger the pipeline automatically by the poll?

Reply:
… automatically by the ~poll~ pull.

Reply:
We’ve explored build scheduling through GitLab, and of course there are other ways. Triggering from Travis looks to be the simplest by far.

Reply:
When you trigger a pull via HTTP POST, shouldn't the new commits automatically trigger a build?